dstep/translator/objc/ObjcInterface.d

```diff
@@ -64,11 +64,11 @@ class ObjcInterface (Data) : Declaration
         output.output(currentClass.data());
     }
 
-    protected string[] collectInterfaces (ObjcCursor cursor)
+    protected string[] collectInterfaces (ObjcCursor obj_cursor)
```

Comment:
Same here, camel casing. And perhaps name it `objcCursor`. “objc” is the standard abbreviation of Objective-C.